modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/tasks/WatchTask.java

```diff
@@ -249,10 +255,13 @@ private static long _getBundleId(
 
 				BundleDTO bundleDTO = _parseBundleDTO(gogoLine);
 
-				if ((bundleDTO != null) &&
-					bundleSymbolicName.equals(bundleDTO.symbolicName)) {
+				if (bundleDTO != null) {
+					String symbolicName = bundleDTO.symbolicName.substring(
+						0, bundleDTO.symbolicName.lastIndexOf("(") - 1);
```

Comment:
Should this be `firstIndexOf` . what is the values of the bundleSymbolicName that you are trying to support now with fragments?

Reply:
The value of bundleSymbolicName is "com.liferay.login.web" and the the value of bundleDTO.symbolicName is "com.liferay.login.web (3.0.4)"

Reply:
It seems there is no firstIndexOf() method exist for String, maybe I can use IndexOf() instead of lastIndexOf(), I will try that